common/src/types/network_order.rs

```diff
@@ -116,16 +106,6 @@ impl NetworkOrder {
     pub fn transition_matched(&mut self, by_local_node: bool) {
         self.state = NetworkOrderState::Matched { by_local_node };
     }
-
-    /// Transitions the state of an order to `Cancelled`
-    pub fn transition_cancelled(&mut self) {
-        self.state = NetworkOrderState::Cancelled;
```

Comment:
given that `transition_cancelled` wasn't used anywhere, should we remove the `NetworkOrderState::Cancelled` variant? A quick grep shows that it isn't set anywhere